### ISSUE TITLE
Re-Order rows using ngrid's reorder feature

### ISSUE THREAD

**Issue by shlomiassaf** [OpenSlides/OpenSlides#4953]
Hi @tsiegleauq 

I'm the author of NGrid which you're using in this project.

First, thanks for choosing the grid!

In the [Agenda page](https://demo.openslides.org/agenda/sort) I saw that you are allowing a reorder of the rows using a different page and component. I would like to point out that NGrid has such feature which you can try out in the [home page](https://shlomiassaf.github.io/ngrid/).

If there are issues preventing you from using it, please share.

I will try helping you guys adopting the grid more quickly, as it's important for me as well!

Thanks.

**Comment by tsiegleauq** [comment]
Hi @shlomiassaf, thanks for working on NGrid. Having easy to use tables that support virtual scrolling is pretty awesome!
The [Agenda page](https://demo.openslides.org/agenda/sort/) you found is from a legacy version of our software OpenSlides, that was written in AngularJS and was using Twitter Bootstrap.

We are using NGrid heavily in OpenSlides 3, a demo page can be found [here](https://nightly.demo.openslides.org/)
The OpenSlides 3 equivalent to the agenda sorting can be found [here](https://nightly.demo.openslides.org/agenda/sort-agenda)

Similar to this "sort-agenda"-view, we have a sorting views for categories and motions (namely a "call list"). I would be happy to use NGrid for sorting tasks, however, the critical part is horizontal "tree-like" sorting, which I suppose NGrid does currently not support, If I am not mistaken.
Basically we need a way to create parent and child relationships in sorting lists.

Here is the relevant issue in Angular CDK
https://github.com/angular/components/issues/13796

![image](https://user-images.githubusercontent.com/10233032/63926592-63129c00-ca4c-11e9-9b33-2e0996992811.png)
